autobot/telegram/parser.py

```diff
@@ -697,3 +697,169 @@ def _parse_sticker (self,key:str,val:dict) -> Sticker|StickerSet:
 
         return sticker_obj
 
+
+    def _parse_animations(self,key:str,val:dict) -> Animation:
+        animation_obj = Animation()
+        for k,v in val.items():
+            match k:
+                case "thumb":
+                    thumb = self._parse_photosize(k,v)
+                    setattr(animation_obj,k,thumb)
+
+                case "file_name"|"mime_type":
+                    setattr(animation_obj,k,v)
+
+                case "file_size":
+                    setattr(animation_obj,k,int(v))
+                case _:
+                    setattr(animation_obj,k,v)
```

Comment:
For example, line 709 to 713 is not needed as it can be done by line 714. So we just delegate all that work there

Reply:
But line 705 does what it's supposed to do because it delegates parsing of `thumb` over to 
`_parse_photosize`